### DISCUSSION TITLE
Floating tile exception for Emote keyboard

### DISCUSSION THREAD

**Discussion by dwoznicki** [pop-os/shell#1391] [Q&A]
I just installed the Emote keyboard app (https://github.com/tom-james-watson/Emote) and would like to add it to the floating window exceptions list. However, when I open up the "Floating Window Exceptions" GUI, the Emote application window automatically closes, which prevents me from selecting it as an exception.

Is there a way I can add floating window exceptions for these sorts of transient applications?

**Comment by AndrewKvalheim** [comment]
What worked for me:

1. In Floating Window Exceptions, click Select.
2. Press the keyboard shortcut for Emote.
3. Click on the Emote window.

The configuration it generated was:

```json
{ "class": "emote" }
```

Perhaps this should be added to [`DEFAULT_FLOAT_RULES`](https://github.com/pop-os/shell/blob/cfa0c55e84b7ce339e5ce83832f76fee17e99d51/src/config.ts#L25).
